tests/ui_tests/app/page/common/popups.py

```diff
@@ -18,14 +18,14 @@
 class CommonPopupLocators:
     """ADCM popup locators"""
 
-    block = Locator(By.XPATH, "//simple-snack-bar", "Popup block")
+    block = Locator(By.XPATH, "//simple-snack-bar", "Common popup block")
```

Comment:
Wouldn't it be confusing to see implementation terminology in a report? Maybe keep here just "Popup block" and it should be understandable from the context of a step.

Reply:
fixed